integreat_cms/cms/models/languages/language.py

```diff
@@ -203,6 +203,24 @@ def can_be_pdf_exported(self) -> bool:
         """
         return self.slug not in settings.PDF_DEACTIVATED_LANGUAGES
 
+    @classmethod
+    def search(cls, query: str) -> QuerySet:
+        """
+        Searches for all languages which match the given `query` in their native name or english name.
+        :param query: The query string used for filtering the languages
+        :return: A query for all matching objects
+        """
+        result = []
+        for language in cls.objects.all():
+            if (
+                query.lower() in language.native_name.lower()
+                or query.lower() in language.english_name.lower()
+                or query.lower() in language.translated_name.lower()
+            ):
+                result += [language.id]
```

Comment:
I know this is not beautiful 🤔 Though, query filter doesn't let users search for `translated_name` of languages as it is `cached_property`, and search not working for translated names may cause confusion because most of users use the system in German (for example "Ungersich". Most users look for "Ungerisch", but its native name is "Magyar" and english name is "Hungarian". Inputs like "Unge..." won't hit the language without taking `translated_name` into account).

Reply:
IMO it's perfectly fine, esp. since there will never be enough languages for this to become a performance issue 😄

Reply:
Since this feature is only available for internal users I don't think we should spend a lot of time on this. For me this is okay too :)